src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <article class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        class="card__image"
+        alt="APPLE-A1419-iMac-27-picture"
+      >
+
+      <h3 class="card__item-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="stars-block">
+        <div class="stars-block__stars stars-block__stars--4">
+          <div class="stars-block__star"></div>
+          <div class="stars-block__star"></div>
+          <div class="stars-block__star"></div>
+          <div class="stars-block__star"></div>
+          <div class="stars-block__star"></div>
+        </div>
+
+        <p class="stars-block__reviewers">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+
+        <p class="card__price-caption">
+          Price:
+        </p>
+
+        <p class="card__price-amount">
+          $2,199
+        </p>
+      </div>
+      <div >
```

Comment:
you didn't fix this small typo
```suggestion
      <div>
```